api.md

```diff
@@ -34,10 +36,12 @@
 * state:element( element_type, data ) - Semi-private, create an element with type and data.
 
 ### Variables
-* state.player - The name of the player.
+* state.player - The name of the current player.
+* state.multiplayer - a list of players connected to the form (on node)
 * state.param - The parameters supplied by form:show.
 * state.def - The form definition.
 * state.is_inv - Boolean which is true if this form is being shown as an inventory.
+* state.nodepos - Node position if attached to node meta
```

Comment:
Agree partily. In case of type "node" there is no player relation. Therefore 
```
{
    type = "node",
    pos = { x = 0, y = 1, z = 2 },
}
```
The viewers should be handled at other place (not static)
Because of less time the merging of "state.player" + "state.nodemeta" to "state.location" is low-prio for me.